### PR TITLE
signup and signin with theme-overrides per designs

### PR DESCRIPTION
**Pull request by Schrese**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by trevorjamesmartin — DISMISSED** [review]

**Review by JLC6290 — DISMISSED** [review]

File: src/components/pages/Home/HomeContainer.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function HomeContainer({ LoadingComponent }) {
         isSubscribed = false;
         return setUserInfo(null);
       });
+    console.log({ authState });
```
Comment (bramirez96) [inline src/components/pages/Home/HomeContainer.js]: Gotta drop this line!

**Review by bramirez96 — CHANGES_REQUESTED** [review]
Just that one small change, but everything else looks great!

**Review by Nick-Ohman — DISMISSED** [review]
very cool way to export, looks good!

**Review by bramirez96 — APPROVED** [review]

**Review by Nick-Ohman — APPROVED** [review]
Great!

**Pull request merged**